java/vector/src/main/java/org/apache/arrow/vector/types/pojo/Schema.java

```diff
@@ -163,6 +160,15 @@ public int getSchema(FlatBufferBuilder builder) {
     return org.apache.arrow.flatbuf.Schema.endSchema(builder);
   }
 
+  public byte[] toByteArray() {
```

Comment:
Is this necessary since you can use the function from MessageSerialzer? Although it does align the bytes
```
ByteArrayOutputStream out = new ByteArrayOutputStream();
MessageSerializer.serialize(new WriteChannel(Channels.newChannel(out)), schema);
out.toByteArray()
```